cmd/zdb/zdb.c

```diff
@@ -2292,6 +2422,8 @@ dump_label(const char *dev)
 			nvlist_t *vdev_tree = NULL;
 
 			dump_nvlist(config, 4);
+			if (l == 3 && dump_opt['l'] >= 2)
```

Comment:
Yeah, good point.  I could  clone the first config nvlist encountered and post process it to get the desired effect of having this stat dump be last.  Is that acceptable?